### PR TITLE
dbconnector&backup script

### PR DESCRIPTION
**Pull request by QwertyJack**
1. mv all sql_query(set utf8) to dbconnector
2. backup&scp scripts

### CODE REVIEW

File: bin/conf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+mysql_passwd=19951025
+database=capubbs
+
+# DB dump dir default, better use absolute path
+dumpdir=./
+
+# backup server
+server_ip=123.131.52.108
+server_user=capu
+server_passwd=
```
Comment (huxuan) [inline bin/conf]: I don't think we need `passwd` here. Please remove this line.

File: bin/conf [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# All config is here !!!
+#
+# By Jack @ 2015-03-23
+
+# working directory
+wd=/work/capu/CAPUBBS
```
Comment (huxuan) [inline bin/conf]: I don't think this `wd` is correct assigned. Can you help me to confirm it works?

File: bin/conf [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+# uesr on this websit, using to crontab
```
Comment (huxuan) [inline bin/conf]: s/websit/website/
s/using to crontab/using crontab/

File: bin/conf (hunk 2 above)
Comment (QwertyJack) [inline bin/conf]: The same.
It's a soft link.

Comment (huxuan) [inline bin/conf]: What I am concern is whether user `capu` has permission to access `/work` directly.

**Pull request merged**